src/frontend/src/pages/Login.tsx

```diff
@@ -22,7 +22,7 @@ interface Props {
 
 export default function Login({ setPages, user, setUser }: Props) {
   const navigate = useNavigate();
-  const authToken = '4f47674fd811e88ce92bcdb21d778e4914b4d903';
```

Comment:
This change doesn't really matter. @Dylando22 should have a fix that we don't even have to worry about this anymore